### PR TITLE
renamed reserved word and referenced objects.

### PR DESCRIPTION
**Pull request by davidpesce**
This should fix issue #16

### CODE REVIEW

**Comment by davidpesce** [comment]
Don't merge this just yet. I found another reference.

**Comment by davidpesce** [comment]
@ryansmith94 - ok, ready for review.

File: src/StatementBase.php [hunk 1]
```diff
@@ -5,36 +5,36 @@ class StatementBase extends Element {
   protected $props = [
     'actor' => 'Locker\XApi\Actor',
     'verb' => 'Locker\XApi\Verb',
-    'object' => 'Locker\XApi\Object',
+    'statementobject' => 'Locker\XApi\StatementObject',
```
Comment (ryasmi) [inline src/StatementBase.php:8]: I don't think this should have changed. There are other references to this here that I don't think should have changed.
```suggestion
    'object' => 'Locker\XApi\StatementObject',
```

File: src/SubStatement.php [hunk 2]
```diff
@@ -21,7 +21,7 @@ public function validate() {
     $errors = parent::validate();
     $errors = array_merge($errors, $this->validateTypedElement());
 
-    $object_type = $this->getPropValue('object.objectType');
+    $object_type = $this->getPropValue('statementobject.objectType');
```
Comment (ryasmi) [inline src/SubStatement.php:24]:
```suggestion
    $object_type = $this->getPropValue('object.objectType');
```

File: tests/assets/a1.json [hunk 3]
```diff
@@ -11,7 +11,7 @@
             "en-US":"created"
         }
     },
-    "object":{
+    "statementobject":{
```
Comment (ryasmi) [inline tests/assets/a1.json:14]: Again, I think all these references in the assets need to be changed back.
```suggestion
    "object":{
```

**Review by ryasmi — CHANGES_REQUESTED** [review]

**Comment by davidpesce** [comment]
@ryansmith94 - what's the best way to test this? I tried updating composer to include phpunit, but the tests never ran.

File: src/StatementBase.php (hunk 1 above)
Comment (davidpesce) [reply src/StatementBase.php:8]: see #18

File: src/SubStatement.php (hunk 2 above)
Comment (davidpesce) [reply src/SubStatement.php:24]: see #18

File: tests/assets/a1.json (hunk 3 above)
Comment (davidpesce) [reply tests/assets/a1.json:14]: see #18

**Comment by ryasmi** [comment]
Closed as replaced by #18

**Pull request closed without merging**